building-view-counter/index050.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<!--
+    Now we are all good with html code, we can work on the javascript code to
```

Comment:
This step could be removed. Instead, learners could see an empty JS file with commented out instructions.